avogadro/mainwindow.cpp

```diff
@@ -522,26 +522,30 @@ void MainWindow::dragEnterEvent(QDragEnterEvent* event)
   else
     event->ignore();
 }
-
 void MainWindow::dropEvent(QDropEvent* event)
 {
-  if (event->mimeData()->hasUrls()) {
-    // TODO: check for ZIP, TAR, PY scripts (plugins)
-    foreach (const QUrl& url, event->mimeData()->urls()) {
-      if (url.isLocalFile()) {
-        QString fileName = url.toLocalFile();
-        QFileInfo info(fileName);
-        QString extension = info.completeSuffix(); // e.g. .tar.gz or .pdb.gz
+  const QMimeData* mimeData = event->mimeData();
 
-        if (extension == "py")
-          addScript(fileName);
-        else
-          openFile(fileName);
-      }
+  if (mimeData->hasUrls()) {
```

Comment:
I mean, that's okay if people drag a file to open (e.g., SDF) but it breaks drag-python-script to install: `addScript(fileName)` above.